webapps/model-fairness-view/backend.py

```diff
@@ -0,0 +1,65 @@
+import dataiku
+import logging
+import simplejson
+import traceback
+from dku_model_accessor import get_model_handler, ModelAccessor
+from dku_webapp import remove_nan_from_list, convert_numpy_int64_to_int, get_metrics, get_histograms
+
+logger = logging.getLogger(__name__)
+
+@app.route("/get-value-list/<model_id>/<version_id>/<column>")
+def get_value_list(model_id, version_id, column):
```

Comment:
the webapp crash when too many values for the selected column, maybe restrict columns to categorical (and numerical with not many distinct values)

Reply:
Done in 4848124